terminus-ui/autocomplete/src/autocomplete.component.ts

```diff
@@ -1,67 +1,101 @@
 // NOTE: A method must be used to dynamically format values for the UI
 // tslint:disable: template-no-call-expression
+import { ActiveDescendantKeyManager } from '@angular/cdk/a11y';
 import {
+  AfterContentInit,
   AfterViewInit,
   ChangeDetectionStrategy,
   ChangeDetectorRef,
   Component,
+  ContentChildren,
   ElementRef,
   EventEmitter,
   Input,
   isDevMode,
   OnDestroy,
+  OnInit,
+  Optional,
   Output,
+  QueryList,
+  Self,
   ViewChild,
   ViewEncapsulation,
 } from '@angular/core';
 import {
   FormControl,
-  ValidationErrors,
+  NgControl,
 } from '@angular/forms';
+import { MatAutocompleteSelectedEvent } from '@angular/material/autocomplete';
+
 import {
-  MatAutocomplete,
-  MatAutocompleteSelectedEvent,
-  MatAutocompleteTrigger,
-} from '@angular/material/autocomplete';
+  BehaviorSubject,
+  of,
+  Subject,
+} from 'rxjs';
+
+import { CdkConnectedOverlay } from '@angular/cdk/overlay';
+import { MatChipList } from '@angular/material';
 import {
-  arrayContainsObject,
+  hasRequiredControl,
   isFunction,
+  TsDocumentService,
   untilComponentDestroyed,
 } from '@terminus/ngx-tools';
+import { coerceNumberProperty } from '@terminus/ngx-tools/coercion';
+import { TsFormFieldControl } from '@terminus/ui/form-field';
 import {
-  coerceArray,
-  coerceNumberProperty,
-} from '@terminus/ngx-tools/coercion';
+  TS_OPTION_PARENT_COMPONENT,
+  TsOptgroupComponent,
+  TsOptionComponent,
+} from '@terminus/ui/option';
 import { TS_SPACING } from '@terminus/ui/spacing';
 import { TsStyleThemeTypes } from '@terminus/ui/utilities';
-import { BehaviorSubject } from 'rxjs';
 import {
-  debounceTime,
-  distinctUntilChanged,
-  filter,
+  debounceTime, distinctUntilChanged, filter, switchMap,
 } from 'rxjs/operators';
+import {
+  TsAutocompletePanelComponent, TsAutocompletePanelSelectedEvent,
+} from './autocomplete-panel/autocomplete-panel.component';
+import { TsAutocompleteTriggerDirective } from './autocomplete-panel/autocomplete-trigger.directive';
```

Comment:
There are some extra line breaks above, and we're missing a line break between 3rd party and internal imports